packages/common/src/types/node.types.ts

```diff
@@ -1 +1 @@
-export const ELEMENT_DEFAULT = 'p';
+export const ELEMENT_DEFAULT = 'div';
```

Comment:
```suggestion
export const ELEMENT_DEFAULT = 'p';
```
As discussed, ELEMENT_DEFAULT is ELEMENT_PARAGRAPH. This variable was defined to avoid adding slate-plugins-paragraph dependency.

Reply:
Done